cypress/integration/select_server.spec.ts

```diff
@@ -0,0 +1,77 @@
+describe('Select Server Page Tests', () => {
```

Comment:
For `describe` and `it`, in both Cypress and Jest, I think we should keep a sane standard.

I think having it build a coherent sentence like what is done in the BlurhashCanvas tests is the best way of writing these.

So like: `This thing - does this` or `This thing - has this expected result`.

So here, I would split this in two:
```typescript
describe('The index page', () => {
  it('redirects to the select server page if the user is not logged in and there is no default server', () => {});
});

describe('The server selection page', () => {
  it('disables the Connect button if no user information is supplied', () => {});
});
```

This properly conveys the scope and intent of each test, and helps other people understand what the test is doing at a glance.